src/output-json.c

```diff
@@ -1096,6 +1180,31 @@ OutputInitResult OutputJsonInitCtx(ConfNode *conf)
                 FatalError("Invalid JSON output option: %s", output_s);
         }
 
+#ifdef HAVE_GEOIP
+        const ConfNode *geoip_enrichment = ConfNodeLookupChild(conf, "geoip-enrichment");
+        if (geoip_enrichment && geoip_enrichment->val && ConfValIsFalse(geoip_enrichment->val)) {
+            SCLogConfig("GeoIP enrichment is disabled.");
+        } else {
+            const char *geoip_db_s = NULL;
+
+            SCLogConfig("GeoIP enrichment is enabled.");
+            (void)ConfGet("geoip-database", &geoip_db_s);
+            if (geoip_db_s == NULL) {
+                mmdb_status = MMDB_FILE_OPEN_ERROR;
+                SCLogWarning("Set geoip-database path if you wish to enable geoip-enrichment");
```

Comment:
suggest something like: `geoip-database` should be set for `geoip-enrichment` functionality